README.md

```diff
@@ -95,19 +97,21 @@ For ClamAV, macOS users can do the following:
 - `source ~/.bash_profile`
 - `freshclam` (installs virus definitions)
 
-Both the front-end (`/web-client`) and API (`/web-api`) share code that exists in `/shared`. Before you can run either, you need to run `npm install` inside the `shared` directory.
```

Comment:
I've noticed that the tabs are difficult to navigate by keyboard, because the tabs do not have any visual indicator when they are focused. For example, the `Messages` tabbed interface:

![messages](https://user-images.githubusercontent.com/656758/63195650-33719600-c041-11e9-9902-8e6ccedefa45.png)

This screenshot is from when `Complete` is focused, but there's no visual indication that is so.

This is true for the higher-level tabs, as well (e.g., "Inbox" and "Sent" in "My Messages"). Unless there is some reason why this was a deliberate design decision, would you please have these behave the same as the top-level menu items (e.g. `Messages | Document QC | Trial Session`), which handles this properly?

Reply:
@waldoj The "fix" for this was very subtle. IIRC, there was a task a while back to make a change to these that likely removed that tab state. I've added a `:focus` state that is essentially the same as the hover state. I don't think it's as clear as the dotted box around the text, but you can actually tell you're on the button now. Let me know if we need to revisit this.